src/module.c

```diff
@@ -1874,6 +1874,13 @@ NAPI_METHOD(dcn_contact_get_status) {
   NAPI_RETURN_AND_UNREF_STRING(status);
 }
 
+NAPI_METHOD(dcn_contact_get_last_seen) {
+  NAPI_ARGV(1);
+  NAPI_DC_CONTACT();
+  int timestamp = dc_contact_get_last_seen(dc_contact);
+  NAPI_RETURN_INT32(timestamp);
```

Comment:
i am wondering about INT32 here, however, as also other timestamps use that type, it seems to be okay :)

Reply:
probably ok until 2038, before that we need to change it.